examples/teleport-usage/teleport_usage.py

```diff
@@ -0,0 +1,115 @@
+import json
+import datetime
+import boto3
+import time
+import os
+
+from boto3.dynamodb.conditions import Key, Attr
+
+
+# Check for AWS DynamoDB table name and region ENV, if none prompt
+TABLE_NAME = os.getenv("TABLE_NAME")
+if TABLE_NAME == None:
+    TABLE_NAME = input("DynamoDB events table name: ")
+
+aws_region = os.getenv("AWS_REGION")
+if aws_region == None:
+    aws_region = input("AWS Region for DynamoDB: ")
+
+# Check ENV for start and end times, if none, set last 30 days
+start_date = os.getenv("START_DATE")
+if start_date == None:
+    now = datetime.datetime.now()
+    start_date = now - datetime.timedelta(days=30)
+    
+end_date = os.getenv("END_DATE")
+if end_date == None:
+    end_date = now
+
+# Configure DynamoDB for query
+dynamodb = boto3.resource("dynamodb", region_name=aws_region)
+table = dynamodb.Table(TABLE_NAME)
+
+# variables for protocols
+app = 'app.session.chunk'
+db = 'db.session.start'
+kube = 'kube.request'
+ssh = 'session.start'
+win = 'windows.desktop.session.start'

```

Comment:
of these vars, only `ssh` is actually referenced.

Reply:
The goal was to eventually allow the user to input what protocols they wanted so the query did not have to run for unused protocols. Did not get to that yet but will in a future version.